src/main/java/com/iota/iri/service/dto/FindTransactionsResponse.java

```diff
@@ -2,6 +2,13 @@
 
 import java.util.List;
 
+/**
+ * This class represents the core API request 'findTransactions'.
+ * Find the transactions which match the specified input and return. 
+ * All input values are lists, for which a list of return values (transaction hashes), in the same order, is returned for all individual elements. 
+ * The input fields can either be bundles, addresses, tags or approvees. 
```

Comment:
in some sense, we still have 2 places to update the docs in:
for example, assume that we decide that you can also find transactions based on a new field, the docs would need to be updated in a few places - API.java & FindTransactionsResponse.java (class def + `getHashes()`)